strings.txt

```diff
@@ -17040,3 +17040,786 @@
+    es = Error de inicio de sesión.
+    ro = Eroare de conectare.
+    pl = Błąd logowania.
+    sk = Pri prihlasovaní sa vyskytla chyba.
+    it = Errore accesso.
+    fr = Erreur de connexion.
+    ja = ログインエラー。
+
+  [editor_login_failed_dialog]
+    tags = ios, android
+    ru = Войти не удалось
+    en = Login failed
+    tr = Giriş başarısız
+    sv = Inloggningen misslyckades
+    nb = Innloggingen mislyktes
+    ko = 로그인 실패
+    id = Masuk gagal
+    hu = Sikertelen bejelentkezés
+    de = Login fehlgeschlagen
+    fi = Kirjautuminen epäonnistui
+    nl = Inloggen mislukt
+    da = Login mislykkedes
+    cs = Přihlašování selhalo
+    zh-Hans = 登录失败
+    zh-Hant = 登錄失敗
+    ar = فشل تسجيل الدخول
+    th = การล็อกอินล้มเหลว
+    uk = Помилка входу
+    pt = O início de sessão falhou
+    es = Error al iniciar sesión
+    ro = Conectare eșuată
+    pl = Nieudane logowanie
+    sk = Prihlasovanie bolo neúspešné
+    it = Accesso fallito
+    fr = La connexion a échoué
+    ja = ログインに失敗しました
+
+  [editor_username_error_dialog]
+    tags = ios, android
+    ru = Неверное имя
+    en = Username is invalid
+    tr = Kullanıcı adı geçersiz
+    sv = Användarnamnet är ogiltigt
+    nb = Brukernavnet er ugyldig
+    ko = 잘못된 사용자 이름
+    id = Nama pengguna tidak benar
+    hu = Érvénytelen felhasználónév
+    de = Benutzername ist ungültig
+    fi = Käyttäjätunnus on virheellinen
+    nl = Gebruikersnaam is ongeldig
+    da = Ugyldigt brugernavn
+    cs = Uživatelské jméno není platné
+    zh-Hans = 用户名不存在
+    zh-Hant = 用戶名不存在
+    ar = اسم المستخدم غير صحيح
+    th = ชื่อผู้ใช้ไม่ถูกต้อง
+    uk = Ім'я користувача є недійсним
+    pt = O nome de utilizador é inválido
+    es = Usuario no válido
+    ro = Nume de utilizator nevalid
+    pl = Nieprawidłowa nazwa użytkownika
+    sk = Používateľské meno je neplatné
+    it = Nome utente non valido
+    fr = Le nom d'utilisateur est invalide
+    ja = ユーザー名が無効です
+
+[editor_place_edited_dialog]
+    tags = ios, android
+    ru = Вы отредактировали объект
+    en = You’ve edited an object!
+    tr = Bir objeyi düzenlediniz'!
+    sv = Du har redigerat ett objekt!
+    nb = Du har redigert et objekt!
+    ko = 개체를 편집했습니다!
+    id = Anda telah mengedit sebuah obyek!
+    hu = Szerkesztettél egy objektumot!
+    de = Sie haben ein Objekt bearbeitet!
+    fi = Olet muokannut kohdetta!
+    nl = Je hebt een object bewerkt!
+    da = Du har redigeret et objekt!
+    cs = Upravili jste objekt!
+    zh-Hans = 您已编辑了一个对象！
+    zh-Hant = 您已編輯了一個對象！
+    ar = لقد قمت بتعديل هدف!
+    th = คุณได้แก้ไขวัตถุ!
+    uk = Ви відредагували об’єкт!
+    pt = Editou um objeto!
+    es = ¡Has editado un objeto!
+    ro = Ați editat un obiect!
+    pl = Obiekt był edytowany!
+    sk = Práve ste upravili objekt!
+    it = Hai modificato un oggetto!
+    fr = Vous avez modifié un objet !
+    ja = オブジェクトを編集しました!
+
+  [editor_login_with_osm]
+    tags = ios, android
+    ru = Войти через OpenStreetMap
```

Comment:
Я вижу в тексте. А в UI где применяются эти строчки, и почему так отличается?

> On Mar 15, 2016, at 11:26, therearesomewhocallmetim <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/2221#discussion_r56128499>:
> 
> > +    zh-Hant = 您已編輯了一個對象！
> > +    ar = لقد قمت بتعديل هدف!
> > +    th = คุณได้แก้ไขวัตถุ!
> > +    uk = Ви відредагували об’єкт!
> > +    pt = Editou um objeto!
> > +    es = ¡Has editado un objeto!
> > +    ro = Ați editat un obiect!
> > +    pl = Obiekt był edytowany!
> > +    sk = Práve ste upravili objekt!
> > +    it = Hai modificato un oggetto!
> > +    fr = Vous avez modifié un objet !
> > +    ja = オブジェクトを編集しました!
> > +
> > +  [editor_login_with_osm]
> > +    tags = ios, android
> > +    ru = Войти через OpenStreetMap
> Единственное отличие — тут название сайта, а там адрес.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/mapsme/omim/pull/2221/files#r56128499 <https://github.com/mapsme/omim/pull/2221/files#r56128499>